internal/cfgplugins/policyforwarding.go

```diff
@@ -256,6 +260,106 @@ func InterfacelocalProxyConfig(t *testing.T, dut *ondatra.DUTDevice, a *attrs.At
 
 }
 
+// InterfacelocalProxyConfigScale configures local-proxy-arp on multiple subinterfaces.
+// When the device does not support the OpenConfig path, vendor-specific CLI commands
+// are applied.
+func InterfacelocalProxyConfigScale(t *testing.T, dut *ondatra.DUTDevice, sb *gnmi.SetBatch, params OcPolicyForwardingParams) *gnmi.SetBatch {
+	if deviations.LocalProxyOCUnsupported(dut) {
+		switch dut.Vendor() {
+		case ondatra.ARISTA:
+			var interfacelocalproxy strings.Builder
+			for _, a := range params.Interfaces {
+				if a.IPv4 != "" {
+					interfacelocalproxy.WriteString(fmt.Sprintf(`
+					interface %s.%d
+					ip local-proxy-arp 
+					!`, params.AggID, a.Subinterface))
+				}
+			}
+			helpers.GnmiCLIConfig(t, dut, interfacelocalproxy.String())
+		default:
+			t.Logf("Unsupported vendor %s for native command support for deviation 'local-proxy-arp'", dut.Vendor())
+		}
+		return nil
+	} else {
+		ocInterface := &oc.Interface{}
+		for i, a := range params.Interfaces {
+			s := ocInterface.GetOrCreateSubinterface(a.Subinterface)
+			b4 := s.GetOrCreateIpv4()
+			parp := b4.GetOrCreateProxyArp()
+			parp.SetMode(oc.E_ProxyArp_Mode(3))
+			gnmi.BatchUpdate(sb, gnmi.OC().Interface(a.Name).Subinterface(uint32(i)).Config(), s)
+		}
+		return sb
+	}
+}
+
+// InterfaceQosClassificationConfigScale configures qos-classification on multiple subinterfaces.
+// When the device does not support the OpenConfig path, vendor-specific CLI commands
+// are applied.
+func InterfaceQosClassificationConfigScale(t *testing.T, dut *ondatra.DUTDevice, sb *gnmi.SetBatch, params OcPolicyForwardingParams) *gnmi.SetBatch {
+	if deviations.QosClassificationOCUnsupported(dut) {
+		switch dut.Vendor() {
+		case ondatra.ARISTA:
+			var interfaceqosconfig strings.Builder
+			for _, a := range params.Interfaces {
+				interfaceqosconfig.WriteString(fmt.Sprintf(`
+				interface %s.%d
+				service-policy type qos input af3 
+				!`, params.AggID, a.Subinterface))
```

Comment:
Same comment as above regarding fmt.Fprintf(..)

https://pkg.go.dev/fmt#Fprintf


```suggestion
				fmt.Fprintf(interfaceqosconfig, `
				interface %s.%d
				  service-policy type qos input af3 
				!`, params.AggID, a.Subinterface)
```